pkg/remote/trans/nphttp2/stream.go

```diff
@@ -106,7 +110,11 @@ func (s *stream) SendMsg(m interface{}) error {
 	ri := rpcinfo.GetRPCInfo(s.ctx)
 
 	msg := remote.NewMessage(m, s.svcInfo, ri, remote.Stream, remote.Client)
```

Comment:
If you pass an empty svcInfo into NewMessage, there might be some panic risk.

Reply:
@jayantxie I'll set the default svcInfo as you commented, then there would be no more worry about passing nil svcInfo to NewMessage.

Reply:
@jayantxie I'll set the default service when svc == nil because svcInfo is different from the service client calls in the original logic. Then there would be no more worry about passing nil service info to NewMessage.

Reply:
I return an error when svcInfo is empty